### DISCUSSION TITLE
Como vocês acompanham o fórum do frontendbr?

### DISCUSSION THREAD

**Discussion by mariorodeghiero** [frontendbr/forum#1287] [Discussão]
Eu estava acompanhando o fórum só por e-mail mesmo, mas como eram muitos e-mails, decidi fazer um app com ElectronJS e reactJS, ainda falta alguns detalhes e também a parte de adicionar o repositório pesquisado ao db do electron. 
Mas queria ter uma ideia de como vocês acompanham, assim posso melhorar o App e assim que eu finalizar a documentação, compartilho o repositório aqui na issue para quem quiser contribuir também.

GitHub Issue: 

![git-issue](https://user-images.githubusercontent.com/24671133/48907228-f7145f00-ee4d-11e8-8000-a21130cc25d0.gif)

**Comment by leandrocunha** [comment]
Eu acompanho por email mesmo, mas a ideia do app é muito boa. Parabéns!

**Comment by leandrocunha** [comment]
Aliás poderia virar uma agregador de repos, tipo um client com varios repos, pq a galera com certeza acompanha mais de um

**Comment by mariorodeghiero** [comment]
A ideia é você pesquisar qualquer repositório como mostra no gif e clicando o "+"você vai adicionar a sua Library. 
Já terminei a parte da pesquisa e criação do local-storage, porque o electron nao têm. Agora só preciso criar a função para salvar na library, hoje a noite devo terminar.

**Comment by mksanti** [comment]
Utilizo o [Gitify](https://github.com/manosim/gitify). Basicamente ativo para receber todas as notificações de determinado repositório e dai td issue aberta ou comentada fico sabendo.

**Comment by didiraja** [comment]
Eu acompanho pelo navegador mesmo, como um fórum qualquer

**Comment by felipefialho** [comment]
Sou crazy, então meu Github fica sempre aberto, além disso, também sou notificado de todas as issues novas.

Um fato curioso, é que fora o Twitter, eu não uso redes sociais. Então acabo usando o próprio Github com essa finalidade e vira e mexe, esbarro em issues do frontendbr, hahaha

**Comment by mariorodeghiero** [comment]
@mikejavier eu vi esse app, mas não cheguei a instalar no pc. A issue você visualiza e comenta no proprio App mesmo?

**Comment by theandersonn** [comment]
Uai! Tbm fico com o GitHub aberto o tempo todo, achei que todo mundo fazia isso! kkk.
Parabéns pelo App!

**Comment by mariorodeghiero** [comment]
@theandersonn  hehe...valeu! Eu ja estava ficando louco de tanto e-mail, mas não queria deixar de acompanhar. 
Assim que tiver uma versão estável do App, vou compartilhar aqui.

**Comment by mksanti** [comment]
@mariorodeghiero, quando vc clica na issue, PR, etc  ele abre no navegador

**Comment by ViniciusGularte** [comment]
Sempre to com github aberto também hehe, e meio que ir da uma olhada no forum ja ficou parte da rotina.

**Comment by allansduarte** [comment]
@mariorodeghiero Parabéns pela aplicação. Eu sou como o @LFeh fico com o Github aberto e só uso o Twitter e LinkedIn.

**Comment by mariorodeghiero** [comment]
valeu @allansduarte, espero disponibilizar a primeira versão do App no inicio da semana, quero me certificar que o auto-update do electron esteja funcionando para cada versão nova do App.

**Comment by kvnol** [comment]
> @lfeh: Sou crazy, então meu Github fica sempre aberto, além disso, também sou notificado de todas as issues novas.

faço das palavras do @lfeh, as minhas

**Comment by alexsandro-xpt** [comment]
Eu não consigo acompanhar... eu tento pelo site do github mas acabo perdendo praticamente todas discussões...

**Comment by andersoonweb** [comment]
Eu sempre quando lembro venho aqui. Mas, essa ideia é extremamente legal. Sugiro colocar uma notificação de quando uma nova issue for aberta.

**Comment by mariorodeghiero** [comment]
@alexsandro-xpt pois é, também acabo perdendo, mas vou tentar deixar um App clean e funcional para acompanhar o forum ou qualquer repositório.

@andersoonweb Sua sugestão é bem legal, já tinha até adicionado como pendência no trello, mas ainda não entendi bem como funciona as notificações da Api do github. 
Possivelmente vou ter que pedir ajuda para contribuidores nessa funcionalidade, ainda sou um mero Junior hehe...

**Comment by andersoonweb** [comment]
boa, @mariorodeghiero ! Abre para a comunidade quando possível, vão te ajudar bastante. Não esqueça sempre em inglês para que "todas" pessoas possam contribuir.
Ah... me veio a cabeça o projeto DevSpace, vale estuda-lo e extrair alguma coisinha.

**Comment by mariorodeghiero** [comment]
Opa @andersoonweb ! 
Consegui criar a primeira versão do App usando local storage com electronjs, testei no Win10 e Mac e funcionou de boa, pensei em abrir outra issue para mostrar, será que é de boa @LFeh ou mantenho só aqui?

Gostaria de dizer também que qualquer contribuição será bem vinda 😃🙏 porque tem coisas que ainda nao encontrei como fazer.

https://github.com/mariorodeghiero/git-issue-react-electronjs

![Git Issue](https://user-images.githubusercontent.com/24671133/49445258-adfecc00-f7b8-11e8-81a0-63240ff2ddff.gif)

**Comment by felipefialho** [comment]
@mariorodeghiero Sugiro colocar lá no [Open-source](https://github.com/frontendbr/open-source) 😄

**Comment by mariorodeghiero** [comment]
Não deste repositório, já coloquei lá. Valeu ✌️

**Comment by andersoonweb** [comment]
massa, @mariorodeghiero eu vou tentar ajudar, mas meu nível tá tão baixo... há tempos parei de estudar a área e estou retornando aos poucos, de acordo com o tempo livre.
Vou tentar tbm instalar no Windows 7. Meu notebook quebrou e estou utilizando no computador.

Parabéns, o projeto tá ANIMAL!

**Comment by alexsandro-xpt** [comment]
Voces conhecem alguma ferramenta pra iOS?

**Comment by mariorodeghiero** [comment]
@alexsandro-xpt Eu não conheço nenhuma não

**Comment by alexsandro-xpt** [comment]
@mariorodeghiero este seu tem release ou preciso clonar o repositório e eu mesmo tenho que compilar uma release?

**Comment by mariorodeghiero** [comment]
@alexsandro-xpt não quiz gerar os binários porque quando gero a release no MAC, notifica no terminal que a vesão gerada é para desenvolvimento e não para compartilhamento, já que meu apple-developer-id não é pago. Então não quiz gerar e disponibilizar com medo de incomodação.
Mas testei clonando e executando no Win10 e MAC e foi de boa, se tiver alguma dificuldade, é só falar.
E em relação ao App para o celular, se manjas um pouco de react-native, poderias copiar o código do projeto e transformalo, já que ele e feito em ReactJS.

**Comment by gregoryfm** [comment]
Uso o DevHub, foi lançado recentemente pelo Bruno.

https://github.com/devhubapp/devhub

Tem app para iOS, Android, Web e a versão desktop está para sair a qualquer momento.

Recomendo bastante.

**Comment by mariorodeghiero** [comment]
pow @gregoryfm vou ver o App, valeu ✌️
